esphome/components/number/__init__.py

```diff
@@ -175,3 +177,36 @@ async def number_set_to_code(config, action_id, template_arg, args):
     template_ = await cg.templatable(config[CONF_VALUE], args, float)
     cg.add(var.set_value(template_))
     return var
+
+
+@automation.register_action(
+    "number.inc",
```

Comment:
Please do change these and update the docs PR
```suggestion
    "number.increment",
```

Reply:
Implemented and documented.